galite-tests/src/test/kotlin/org/kopi/galite/tests/domain/DomainTests.kt

```diff
@@ -127,4 +127,28 @@ class DomainTests {
         assertEquals(1, codes["cde1"])
         assertEquals(2, codes["cde2"])
     }
+
+    /**
+     * Tests Domain of a type code with redundant code.
+     */
+    @Test
+    fun domainRedundantCodeTest() {
+        // Declaration of the domain with codes
+        class IntTestType: Domain<Long>(5) {
+            override val values = code {
+                this["cde1"] = 1
+                this["cde2"] = 2
+                this["cde1"] = 7
```

Comment:
You should use the set method in DomainCode.kt instead.

Reply:
it already uses set method in DomainCode.kt

Reply:
It should be like this         set("cde1", 1)

![image](https://user-images.githubusercontent.com/39685547/90424089-78496600-e0b5-11ea-84ad-3a8e3ffecde6.png)

Reply:
https://kotlinlang.org/docs/reference/operator-overloading.html#indexed